ZenPacks/zenoss/LinuxMonitor/migrate/UpdateFileSystemTransform.py

```diff
@@ -0,0 +1,92 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+"""Update the /Perf/Filesystem transform to work with percentUsed datapoint."""
+
+import hashlib
+import logging
+
+from Products.ZenModel.ZenPack import ZenPackMigration
+from Products.ZenModel.migrate.Migrate import Version
+
+LOG = logging.getLogger("zen.migrate")
+
+NEW_TRANSFORM="""
+def transform():
+    if component and evt.eventKey:
+        try:
+            current = float(evt.current)
+            totalBytes = int(component.totalBytes())
+        except Exception:
+            return
+
+        if totalBytes < 1:
+            return
+
+        usedPercent, usedBytes, freeBytes = None, None, None
+
+        # SNMP (Platform)
+        if 'usedBlocks' in evt.eventKey:
+            usedBytes = int(current) * component.blockSize
+            usedPercent = (usedBytes / totalBytes) * 100.0
+            freeBytes = totalBytes - usedBytes
+
+        # Linux SSH (LinuxMonitor ZenPack)
+        elif 'percentUsed' in evt.eventKey:
+            usedPercent = current
+            usedBytes = totalBytes * (current * 0.01)
+            freeBytes = totalBytes - usedBytes
+
+        # Windows (Windows ZenPacks)
+        elif 'FreeMegabytes' in evt.eventKey:
+            freeBytes = int(current) * 1048576
+            usedBytes = totalBytes - freeBytes
+            usedPercent = (usedBytes / totalBytes) * 100.0
+
+        else:
+            return
+
+        # Make a nicer event summary.
+        from Products.ZenUtils.Utils import convToUnits
+        evt.summary = "disk space threshold: %3.1f%% used (%s free)" % (
+            usedPercent, convToUnits(freeBytes))
+        evt.message = evt.summary
+
+transform()
+""".strip()
+
+ORIGINAL_DIGEST="18c01c20b29fef6f0b097a8bea443fa6"
+
```

Comment:
What happens to the calculation below if the source version of this digest changes?
Minor: Pep8 want spaces around that = sign, and NEW_TRANSFORM= above.